src/main/java/org/betonquest/betonquest/config/Config.java

```diff
@@ -256,45 +257,45 @@ public static void sendMessage(final String packName, final String playerID, fin
      * variables and play the sound. It will also add a prefix to the message.
      *
      * @param packName        ID of the pack
-     * @param playerID        ID of the player
+     * @param profile         the {@link Profile} of the player
      * @param messageName     ID of the message
      * @param variables       array of variables which will be inserted into the message
      * @param soundName       name of the sound to play to the player
      * @param prefixName      ID of the prefix
      * @param prefixVariables array of variables which will be inserted into the prefix
      */
-    public static void sendMessage(final String packName, final String playerID, final String messageName, final String[] variables, final String soundName,
+    public static void sendMessage(final String packName, final Profile profile, final String messageName, final String[] variables, final String soundName,
                                    final String prefixName, final String... prefixVariables) {
-        final String message = parseMessage(packName, playerID, messageName, variables, prefixName, prefixVariables);
+        final String message = parseMessage(packName, profile, messageName, variables, prefixName, prefixVariables);
         if (message == null || message.length() == 0) {
             return;
         }
 
-        final Player player = PlayerConverter.getPlayer(playerID);
+        final Player player = profile.getOnlineProfile().getOnlinePlayer();
         player.sendMessage(message);
         if (soundName != null) {
-            playSound(playerID, soundName);
+            playSound(profile, soundName);
         }
     }
 
-    public static void sendNotify(final String packName, final String playerID, final String messageName, final String category) throws QuestRuntimeException {
-        sendNotify(packName, playerID, messageName, null, category);
+    public static void sendNotify(final String packName, final Profile profile, final String messageName, final String category) throws QuestRuntimeException {
+        sendNotify(packName, profile, messageName, null, category);
     }
 
     public static void sendNotify(final String packName, final Player player, final String messageName, final String category) throws QuestRuntimeException {
```

Comment:
There are still a lot Player refs. And you also need to remove them in the NotifyIO class